guides/intro-to-ploomber/_source.md

```diff
@@ -0,0 +1,106 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.6
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+
+# Your first Python pipeline
+
+<!-- start description -->
+To run this example locally, [install Ploomber](https://docs.ploomber.io/en/latest/get-started/quick-start.html) and execute: `ploomber examples -n guides/first-pipeline`
+
+Found an issue? [Let us know.](https://github.com/ploomber/projects/issues/new?title=guides/first-pipeline%20issue) Have questions? [Ask us anything on Slack.](https://ploomber.io/community/)
+<!-- end description -->
+
+# Ploomber Tutorial Intro
+
+- This should showcase the Ploomber value proposition. 
+- We'll do a few ML operations in a notebook, using a sample covid-19 dataset. 
+- We'll forcast the relation between testing and active covid cases.
+
+**For a deeper dive**, try the [first-pipeline guide](https://docs.ploomber.io/en/latest/get-started/first-pipeline.html). 
+
+If YAML, Jupyter and notebooks sounds like a distant cousin, please check our [basic concepts guide](https://docs.ploomber.io/en/latest/get-started/basic-concepts.html).
+
+### We'll see today how you can improve your work:
+- Run 100s of notebooks in parallel 
+- Parameterize your workflows
+- Easily generate HTML/PDF reports
+
+
+# Parallelization
+
+- Ploomber creates a pipeline for you, so you can run independent tasks simultanously. 
+
+- It also cache some results so you don't have to wait. You can drop the `force=True` (last line) and rerun this cell.
```

Comment:
this is a bit confusing. to introduce the cache, it's better to first have a cell that calls `build()` then have another one that calls `build()` again and show it didn't run anything

Reply:
Yeah, the main focus here was parallel, not caching, I just mentioned it as a side note.